webhook/index.js

```diff
@@ -42,8 +61,12 @@ router.post('/api/chat/', async function(req, res) {
     };
 
     // INTENTS
+    // Directly send response if paramenters not complete.
+    if(!dialogflowResponse.queryResult.allRequiredParamsPresent){
```

Comment:
queryResult not present within dialogflowResponse. Please refer to page 7 in https://docs.google.com/document/d/1zZWPV-NhhTIftDaDs09BRJbpvnveO736jOR-tkroKBE/edit#bookmark=id.trnsztq794ph